### PR TITLE
Adding that comma seperated lists of indices are supported.

### PR DESCRIPTION
**Pull request by landon-lengyel**
Signed-off-by: Landon Lengyel <landon@almonde.org>

### Description
The _cluster/health API supports comma separated indices (like most endpoints) but that isn't specified. This resolves that.

### Issues Resolved
_List any issues this PR will resolve, e.g. Closes [...]._

### Version
2.16 - Likely all

### Frontend features
_If you're submitting documentation for an OpenSearch Dashboards feature, add a video that shows how a user will interact with the UI step by step. A voiceover is optional._ 

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _api-reference/cluster-api/cluster-health.md [hunk 1]
```diff
@@ -25,6 +25,13 @@ GET _cluster/health
 GET _cluster/health/<index>
 ```
 
+## URL Parameters
```
Comment (kolchfa-aws) [inline _api-reference/cluster-api/cluster-health.md:28]:
```suggestion
## Path parameters

```

File: _api-reference/cluster-api/cluster-health.md [hunk 2]
```diff
@@ -25,6 +25,13 @@ GET _cluster/health
...
+The following table lists the available URL parameters. All URL parameters are optional.
```
Comment (kolchfa-aws) [inline _api-reference/cluster-api/cluster-health.md:29]:
```suggestion
The following table lists the available path parameters. All path parameters are optional.
```

File: _api-reference/cluster-api/cluster-health.md [hunk 3]
```diff
@@ -25,6 +25,13 @@ GET _cluster/health
...
+
+Parameter | Type | Description
```
Comment (kolchfa-aws) [inline _api-reference/cluster-api/cluster-health.md:31]:
```suggestion
Parameter | Data type | Description
```

File: _api-reference/cluster-api/cluster-health.md [hunk 4]
```diff
@@ -25,6 +25,13 @@ GET _cluster/health
...
+:--- | :--- | :---
+&lt;index-name&gt; | String | Limit health reporting to a specific index. Can be a comma-separated list of multiple index names.
```
Comment (kolchfa-aws) [inline _api-reference/cluster-api/cluster-health.md:33]:
```suggestion
&lt;index-name&gt; | String | Limits health reporting to a specific index. Can be a single index or a comma-separated list of index names.
```

**Review by kolchfa-aws — COMMENTED** [review]
Thank you, @landon-lengyel! A couple of suggestions to make this adhere with our [API template](https://github.com/opensearch-project/documentation-website/blob/main/templates/API_TEMPLATE.md)

**Review by natebower — APPROVED** [review]
LGTM

**Pull request merged**